plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/WebSetupStepDefs.java

```diff
@@ -1,15 +1,20 @@
-package ru.sbtqa.tag.pagefactory.web.stepdefs;
+package ru.sbtqa.tag.pagefactory.stepdefs;
 
-import cucumber.api.java.Before;
 import ru.sbtqa.tag.pagefactory.environment.Environment;
 import ru.sbtqa.tag.pagefactory.web.actions.WebPageActions;
 import ru.sbtqa.tag.pagefactory.web.checks.WebPageChecks;
 import ru.sbtqa.tag.pagefactory.web.drivers.WebDriverService;
 
 public class WebSetupStepDefs {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)